packages/release-mastermind/src/contexts/schedule.ts

```diff
@@ -0,0 +1,131 @@
+import * as core from "@actions/core";
+import { Context } from "@actions/github/lib/context";
+import { LoggingDataClass, LoggingLevels } from "@videndum/utilities";
```

Comment:
Codacy has a fix for the issue: [There should be no space after '{'.](https://app.codacy.com/gh/Videndum/action-masterminds/pullRequest?prid=7872005)
```suggestion
import {LoggingDataClass, LoggingLevels } from "@videndum/utilities";
```